packages/nep141-erc20/src/natural-erc20/sendToNear/index.js

```diff
@@ -128,7 +129,8 @@ export async function initiate ({
     completedConfirmations: 0,
     lockHashes: [],
     lockReceipts: [],
-    neededConfirmations: 30 // hard-coding until connector contract is updated with this information
+    neededConfirmations: 30, // hard-coding until connector contract is updated with this information
```

Comment:
The confirmations are dependant on the bridge instance. For mainnet bridge we use 20 confirmations at the moment, less than that for Ropsten. We can change this into 20 as a quick hack.